controllers/grants.go

```diff
@@ -81,6 +84,21 @@ func ClaimGrant(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	conn := datastore.GetRedisConn(r.Context())
+
+	if err == nil {
```

Comment:
`err` here is the error returned from `govalidator.ValidateStruct`. Not sure if this is required.

Reply:
good catch, yeah this is totally redundant - I will remove it